### ISSUE TITLE
GUI is totally unusable

### ISSUE THREAD

**Issue by kuanyui** [eight04/ComicCrawler#20]
- python3-tk-3.4.5
- KDE 5

GUI is totally unusable, which shows nothing. Without any error message
![md8](https://cloud.githubusercontent.com/assets/1370070/19837797/7a9bee86-9efc-11e6-90a7-b38b875ab397.png)
.

**Comment by kuanyui** [comment]
Maybe I can rewrite in PyQt. ...If have free time.

**Comment by eight04** [comment]
Does old version work?

**Comment by eight04** [comment]
PyQt is licensed under GPL which is not compatible with MIT.

**Comment by kuanyui** [comment]
Ah...Q_Q

**Comment by kuanyui** [comment]
> Does old version work?

What did you mean?

**Comment by eight04** [comment]
Can you test if the bug is related to Comic Crawler or tkinter itself? Also, try installing version 2016.6.25:
```
pip install -I comiccrawler==2016.6.25
```

**Comment by kuanyui** [comment]
![md8](https://cloud.githubusercontent.com/assets/1370070/19860936/a3c3b75c-9fc5-11e6-978d-afbef6d35bdc.png)
`2016.6.25` 's GUI is ok.

**Comment by eight04** [comment]
I guess it is related to scaling or linux theme.

Try removing these lines:
https://github.com/eight04/ComicCrawler/blob/master/comiccrawler/gui.py#L288-L292

**Comment by kuanyui** [comment]
Sorry, I still cannot figure out how to run this project for development purpose. (I'm very unfamiliar with Python's modularization ways...)

I created `run.py` under project root path:

```python
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
import comiccrawler
comiccrawler.console_init()
```

But execute with `run.py gui`, it said:

```python
Traceback (most recent call last):
  File "./run.py", line 5, in <module>
    comiccrawler.console_init()
  File "/home/kuanyui/tmp/ComicCrawler/comiccrawler/__init__.py", line 48, in console_init
    from .gui import MainWindow
  File "/home/kuanyui/tmp/ComicCrawler/comiccrawler/gui.py", line 27, in <module>
    from .download_manager import download_manager
  File "/home/kuanyui/tmp/ComicCrawler/comiccrawler/download_manager.py", line 8, in <module>
    from worker import Worker, current, later
ImportError: cannot import name 'later'
```

**Comment by eight04** [comment]
After cloning the repo:

1. Go to root folder (where setup.py located) and run `pip install -e .`. It will install the package in the folder.
2. Execute the module within `python -m comiccrawler` command. (`python -m comiccrawler gui` to launch GUI window)
3. If you like to install other dependencies for development purpose, run `pip install -r requirements.txt`

References:
https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs
https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-r

**Comment by eight04** [comment]
BTW, the import error is caused by incompatible version of `pythreadworker`.

Currently we use pythreadworker 0.6. But v2016.6.25 was came with pythreadworker 0.5.

**Comment by kuanyui** [comment]
>I guess it is related to scaling or linux theme.

>Try removing these lines:
>https://github.com/eight04/ComicCrawler/blob/master/comiccrawler/gui.py#L288-L292

not works. Problem remains.

**Comment by eight04** [comment]
Then try remove these lines:
https://github.com/eight04/ComicCrawler/blob/master/comiccrawler/gui.py#L302-L304
or
https://github.com/eight04/ComicCrawler/blob/master/comiccrawler/gui.py#L306-L312

**Comment by kuanyui** [comment]
>Then try remove these lines:
https://github.com/eight04/ComicCrawler/blob/master/comiccrawler/gui.py#L302-L304
or
https://github.com/eight04/ComicCrawler/blob/master/comiccrawler/gui.py#L306-L312

Not works neither.

**Comment by kuanyui** [comment]
Hmm... I found the point. `get_scale(self.root)` return `0.0` ....

**Comment by eight04** [comment]
👍 
Would you like to do a PR? Or I will fix it these days.

**Comment by kuanyui** [comment]
I have to survey how to get screen density on KDE....

**Comment by eight04** [comment]
Related issue:
https://github.com/eight04/ComicCrawler/issues/13

I will do a temporary fix: `scale = 1.0 if scale < 1 else scale`

**Comment by kuanyui** [comment]
Ok I found how to get it in KDE5:
in `~/.config/kdeglobals`
```ini
[KScreen]
ScaleFactor=1
```
I will find time to throw PR.

**Comment by eight04** [comment]
Fixed in 2016.11.2.
